### ISSUE TITLE
Limitation of 256 Layers

### ISSUE THREAD

**Issue by haxekhaex2** [drawpile/Drawpile#1148]
A limitation of 256 layers makes extended animation hard.

**Comment by askmeaboutlo0m** [comment]
Yeah, probably not something that will change before Drawpile 3.0, since the layer limit is part of the protocol.

What could be done meanwhile is to allow operators to create extra layers, since they're technically allowed to do so. It will kind of break layer permissions, but nobody's gonna assign permissions to hundreds of layers individually, so it probably doesn't matter.

**Comment by askmeaboutlo0m** [comment]
Operators can now create layers beyond their own. They will start using layers of user 0 (the server) and once those are used up they'll start filling up layers from user 255 downwards. Which should be fine in most cases, unless your session has hundreds of users.

This also works when drawing offline, since then you also count as being operator.

That's available in the continuous development release: <https://github.com/drawpile/Drawpile/releases/tag/continuous>

**Comment by askmeaboutlo0m** [comment]
This is implemented and will be coming in Drawpile 2.3.0. The hard limit for layers per user will be 32767 per user instead (which will probably get some kinda permission setting to limit it.) Operators continue to be able to use other users' layers, so they could theoretically have even more of them. Thanks for suggesting!
